### ISSUE TITLE
Nokigiri 1.12.5 compatibility

### ISSUE THREAD

**Issue by Davids89** [ueno/ruby-gpgme#151]
Hello everyone, I'm trying to run a project with gpgme version 2.0.20 and nokogiri version 1.12.5 but I have this error

```bash
#35 22.71   In Gemfile:
#35 22.71     gpgme (~> 2.0.18) was resolved to 2.0.20, which depends on
#35 22.71       mini_portile2 (~> 2.3)
#35 22.71
#35 22.71     nokogiri (~> 1.12.5) was resolved to 1.12.5, which depends on
#35 22.71       mini_portile2 (~> 2.6.1)
```

I checked the gemspec and I saw `mini_portile` is set to ~>2.3, can it be updated?

Thanks!

**Comment by dansketcher** [comment]
Hey @Davids89 - happy to have a look at it, if you get a chance can you have a go at putting together a PR with the change presuming the tests are ok?
Ideally we'd keep it compatible with existing installations, not sure on the version dependency range of the top of my head.

**Comment by Davids89** [comment]
https://github.com/ueno/ruby-gpgme/pull/152

**Comment by ueno** [comment]
Should be fixed with #163 and the 2.0.22 release.
